### PR TITLE
Upgrade rspec to 3.4.x

### PR DESCRIPTION
**Pull request by elandesign**
Couple of minor syntax changes, nothing major.

### CODE REVIEW

**Comment by alyssais** [comment]
Why not go to 3.6?

**Comment by elandesign** [comment]
Oh bloody hell, I thought I'd checked and 3.4 was the most recent stable release. 🤦‍♂️ 

Thanks, will bump it again.

File: spec/tom_queue/external_consumer_integration_spec.rb [hunk 1]
```diff
@@ -100,7 +100,7 @@ def self.block_method(work)
     end
 
     describe "if something other than a Delayed::Job instance is returned" do
-      it "should ack the original AMQP message" do
+      xit "should ack the original AMQP message" do
         pending('...')
       end
```
Comment (cfabianski) [inline spec/tom_queue/external_consumer_integration_spec.rb]: Let's just delete it if it does nothing?

File: spec/tom_queue/queue_manager_spec.rb [hunk 2]
```diff
@@ -81,7 +81,7 @@
       end
 
       it "should throw an ArgumentError exception if :run_at isn't a Time object" do
-        lambda { 
+        lambda {
           manager.publish("future", :run_at => "around 10pm ?")
         }.should raise_exception(ArgumentError, /must be a Time object/)
```
Comment (cfabianski) [inline spec/tom_queue/queue_manager_spec.rb]: Wanna try `expect` syntax? 😉

Comment (elandesign) [inline spec/tom_queue/queue_manager_spec.rb]: Dude, it's a whitespace change... I could go through it all and update the syntax if you like, but until it's deprecated then it seems out of scope for a version upgrade.

Comment (cfabianski) [inline spec/tom_queue/queue_manager_spec.rb]: I tried 😜
